### ISSUE TITLE
Very Odd Crash

### ISSUE THREAD

**Issue by TheBrokenRail** [Prospector/ModMenu#150]
When clicking the mods button in dev 20w30a it crashes with:
```
java.lang.ClassCastException: class net.minecraft.class_5481$$Lambda$3262/0x0000000840a17040 cannot be cast to class net.minecraft.text.StringRenderable (net.minecraft.class_5481$$Lambda$3262/0x0000000840a17040 and net.minecraft.text.StringRenderable are in unnamed module of loader net.fabricmc.loader.launch.knot.KnotClassLoader @5f77d0f9)
	at io.github.prospector.modmenu.gui.DescriptionListWidget.render(DescriptionListWidget.java:58)
	at io.github.prospector.modmenu.gui.ModsScreen.render(ModsScreen.java:250)
	at net.minecraft.client.render.GameRenderer.render(GameRenderer.java:603)
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1020)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:653)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)
```

**Comment by TheBrokenRail** [comment]
This also happens in ```1.16-pre2```:
```
java.lang.ClassCastException: class net.minecraft.class_5481$$Lambda$3330/0x0000000840a1f840 cannot be cast to class net.minecraft.text.StringRenderable (net.minecraft.class_5481$$Lambda$3330/0x0000000840a1f840 and net.minecraft.text.StringRenderable are in unnamed module of loader net.fabricmc.loader.launch.knot.KnotClassLoader @37313c65)
	at io.github.prospector.modmenu.gui.DescriptionListWidget.render(DescriptionListWidget.java:58)
	at io.github.prospector.modmenu.gui.ModsScreen.render(ModsScreen.java:250)
	at net.minecraft.client.render.GameRenderer.render(GameRenderer.java:603)
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1020)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:653)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)
```

**Comment by TheBrokenRail** [comment]
Fixed by #151
